ext/SerializationExt.jl

```diff
@@ -29,20 +29,21 @@ function Serialization.deserialize(s::AbstractSerializer, ::Type{LRU{K, V}}) whe
     lock = Serialization.deserialize(s)
     by = Serialization.deserialize(s)
     finalizer = Serialization.deserialize(s)
+    n_items = Serialization.deserialize(s)
```

Comment:
is there a way to detect there are no bytes left to deserialize and fallback to `n_items = currentsize` here? to continue supporting de-serializing old serialized files

Reply:
(if not, I'm not sure this should be a patch release, it seems like it could unexpectedly break stuff. Not quite sure what folks use LRU serialization for though, is it for IPC only or for persistence?)

Reply:
That's a good point; this will break deserializing old serialized data. I think we can make this backward compatible, and that would be nicer in case there are some persisted LRUs out there.

Any existing file would have `n_items` in the `currentsize` position, so we can continue storing `n_items` there, and always recompute `currentsize`.

Reply:
I updated the PR. Thanks for pointing this out!

Reply:
good solution!